presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixInitializer.java

```diff
@@ -26,14 +24,18 @@
 import io.prestosql.plugin.hive.HdfsConfigurationInitializer;
 import io.prestosql.spi.Node;
 import io.prestosql.spi.NodeManager;
+import io.prestosql.spi.PrestoException;
+import net.jodah.failsafe.Failsafe;
+import net.jodah.failsafe.RetryPolicy;
 import org.apache.hadoop.conf.Configuration;
 
+import javax.annotation.PostConstruct;
 import javax.inject.Inject;
 
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
+import java.time.Duration;
 
 import static io.prestosql.plugin.hive.util.ConfigurationUtils.getInitialConfiguration;
+import static io.prestosql.spi.StandardErrorCode.EXCEEDED_TIME_LIMIT;
 
 /*
```

Comment:
We should drop `Lazily` from the comments now as initialisation is now synchronous